misc: Add GPU info to release notes

powerjg opened the pull request.

powerjg committed:
misc: Add GPU info to release notes

Signed-off-by: Jason Lowe-Power <jason@lowepower.com>

abmerop commented on RELEASE-NOTES.md:
```diff
@@ -160,6 +160,22 @@ In this way it will be possible to connect external RNF models to the ruby inter
+
+* Vega10 has been deprecated #1619
+* Replacement policy has been improved #1564
+* Improvements to the TLB #1431
```
I think @mattsinc is planning to revert this with #1769 .

BobbyRBruce replied:
I remove this line based on this comment. Probably best not advertise things that will get reverted.

mattsinc replied:
Thanks @abmerop , I just merged #1769 in after manually making the requested changes.

abmerop reviewed (commented):
Looks like that covers everything that's not a bug fix PR, thanks!

BobbyRBruce committed:
misc: Rm "Improv. to TLB" from v24.1 release notes

According to https://github.com/gem5/gem5/pull/1844/files/5b786e4f1100f77b16a2c69d78990e4c30e1515d#r1874190856, this is going to get reverted anyway.

mattsinc committed:
Update RELEASE-NOTES.md

Update GPU release notes

mattsinc reviewed (approved):
This seems like the major things from my end after my additions!
